content/blog/check_my_repo.md

```diff
@@ -0,0 +1,150 @@
+---
+title: (Ͼ˳Ͽ)...check-my-repo!!!
+date: 2022-01-14T09:00:00+02:00
+description: This is a text about open source, and everything you need to be proud of your repository. Because open standards ensure not only the quality and shareability of a project with necessary documentation to make it welcoming, understandable and keeping an ongoing clear communication. Most importantly, open standards guide technology development by gently enforcing space for diversity and equity.
+author: 'Paloma Oliveira'
+authorTwitterHandle: 'pcultural'
+images:
+    - /images/blog/check_my_repo_steps.png
+tags:
+    - open source
+    - open governance
+    - standards
+    - diversity and equity
+---
+
+This is a text about open source, and everything you need to be proud of your repository. Because open standards ensure not only the quality and shareability of a project with necessary documentation to make it welcoming, understandable and keeping an ongoing clear communication. Most importantly, open standards guide technology development by gently enforcing space for diversity and equity.
```

Comment:
Does this rephrase makes more sense? 

This is a text about open source, and everything you need to be proud of your repository. In the text I argue that by using open standards, an open source project improves its quality and shareability, since such standards exist to foster better communication between creators and consumers of the project. Most importantly, open standards guide technology development by gently enforcing space for diversity and equity.

Reply:
yes